tests/unit/customizations/configure/test_configure.py

```diff
@@ -238,6 +239,19 @@ def test_dont_mask_unset_value(self):
         self.assertEqual(no_config.value, NOT_SET)
 
 
+class TestProfileToSection(unittest.TestCase):
```

Comment:
It would be nice to add a few more tests for cases that aren't just a single space.

Reply:
Did this ever get incorporated?  You mean a test with multiple consecutive space right?  It would be nice to get a test added for that.

Reply:
He meant other valid white space in reference to your previous comment. I added the test for tabs as well. I can add a test for multiple spaces though.